### PR TITLE
Backbone 1.3.2

### PR DESCRIPTION
**Pull request by jridgewell**
Provided there aren't any problems, we'll merge and push this out as v1.3.2.

Ping @megawac, @akre54, @paulfalgout.

Closes https://github.com/jashkenas/backbone/issues/3975.

### CODE REVIEW

**Comment by akre54** [comment]
Run `npm run doc` too?

File: index.html [hunk 1]
```diff
@@ -4297,16 +4297,9 @@ <h2 id="examples-tzigla">Tzigla</h2>
...
-    <ul style="margin-top: 5px;">
-      <li>
-        Fixed some build artifacts with the 1.3.0 release.
-      </li>
-    </ul>
-
-    <b class="header">1.3.0</b> &mdash; <small><i>Mar. 3, 2016</i></small>
-    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.3...1.3.0">Diff</a>
-    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.3.0/index.html">Docs</a>
+    <b class="header">1.3.2</b> &mdash; <small><i>Mar. 4, 2016</i></small>
```
Comment (akre54) [inline index.html]: Make sure to update this after release.

**Comment by akre54** [comment]
:+1:

**Comment by paulfalgout** [comment]
Looks right, but now how about getting this in https://github.com/jashkenas/backbone/pull/3980

**Comment by paulfalgout** [comment]
ping @jridgewell :shipit: ?

**Comment by akre54** [comment]
We good to go @jridgewell ?

**Comment by jridgewell** [comment]
Sorry, had a mid-week vacation. Will get this out tonight.

**Comment by jridgewell** [comment]
Ran redoc and updated the release date. Merging.

**Comment by megawac** [comment]
@jridgewell don't forget to rebase gh-pages in the future

**Pull request merged**